content/academy/tutorials/js_in_html.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: JavaScript objects in HTML
 description: Learn about "hidden" data found within the JavaScript of certain pages, which can increase the scraper reliability and improve your development experience.
-menuWeight: 9
+menuWeight: 2
 paths:
-    - web-scraping-for-beginners/data-collection/js-in-html
+    - tutorials/js-in-html
```

Comment:
I'm not sure about moving this to tutorials. It's an important thing that should be part of some course. Tutorials are for complete guides like "How to scrape with Node.js" or "How to scrape with Playwright". I don't have a strong opinion about it, but I'm afraid we'll end up moving it back into some course later.

Reply:
I moved it because it really didn't fit into the Web Scraping for Beginners course.